tests/backends/base/test_operations.py

```diff
@@ -144,3 +148,46 @@ def test_subtract_temporals(self):
         )
         with self.assertRaisesMessage(NotSupportedError, msg):
             self.ops.subtract_temporals(duration_field_internal_type, None, None)
+
+
+class CommonDatabaseOperationsTests(TestCase):
+    def test_sql_flush_no_tables(self):
+        self.assertEqual(connection.ops.sql_flush(no_style(), [], []), [])
+
+
+class ExecuteSqlFlushTests(TransactionTestCase):
+    available_apps = ['backends']
+
+    def test_execute_sql_flush_statements(self):
+        with transaction.atomic():
+            author = Author.objects.create(name="George Orwell")
+            Book.objects.create(author=author)
+            Book.objects.create(author=author)
+            author = Author.objects.create(name="Harper Lee")
+            Book.objects.create(author=author)
+            author = Author.objects.create(name="J.R.R. Tolkien")
+            Book.objects.create(author=author)
+            Book.objects.create(author=author)
+            Book.objects.create(author=author)
```

Comment:
I don't think that we need 3 authors and 6 books for this test. I will remove the last author.